projects/plugins/jetpack/_inc/client/at-a-glance/backups.jsx

```diff
@@ -102,6 +102,71 @@ class DashBackups extends Component {
 		} );
 	};
 
+	getJetpackBackupBanner() {
+		const { partnerCoupon, upgradeUrl, siteRawUrl, trackUpgradeButtonView } = this.props;
+
+		if ( this.props.hasConnectedOwner ) {
+			if ( partnerCoupon ) {
```

Comment:
I'm not sure how well if fits with the rest of the upgrade boxes saying `Connect your WordPress.com account to upgrade and get automatic backups that keep your content safe.` but we could move this to a `if ( this.props.siteConnected && partnerCoupon )` condition since going through the redeem process (aka checkout) would also create the user connection

Reply:
I am not sure what you are suggesting - could you please provide a bigger example?

Reply:
Sure thing! I'll try and be as specific as possible:

* Create a new site (could be JN)
* Start connecting Jetpack (click one of the `Set up Jetpack` buttons)
* When you reach WPCOM it asks you to `Approve` (connecting your WPCOM user)
* If you go back to your website without approving, your site will be in a connected state, but without a connected user

When we have a connected site, without a connected user, all the upgrade buttons will show an option to "Connect" instead of "upgrade":

![Screenshot 2022-02-03 at 23 16 16](https://user-images.githubusercontent.com/3846700/152438417-f8b9f524-a341-43ab-86d7-8155da6e693a.png)

Technically, we could already start to promote the partner coupon at this stage since the user is redirected to checkout which requires them to connect their user if they haven't already done so.
If we end up redirecting to a more elaborate redeem component, the process remains the same. The user will be redirected to checkout afterwards where Calypso asks them to connect their user.

So we could make a condition above `if ( this.props.hasConnectedOwner )` where we just look for a partnerCoupon _(the at a glance page will never be rendered if the site isn't connected, so we don't even have to check for a site connection which I initially suggested - just a partner coupon)_.

Attempt to visually represent it:
```
if ( partnerCoupon ) {
    return <PartnerCouponBanner>;
} else if ( this.props.hasConnectedOwner ) {
    return <UpgradeBanner>;
}

return <PleaseConnectJetpackBanner>;
```

Reply:
I'd rather we stick to the task. In other cases where a connection is not established, we explicitly change the verbiage to "Set up and Redeem" which we don't do here. Also doing this now makes the Backup Connect CTA differ from the Scan and Anti-spam ones so I could see a question like the following pop in customers' minds:
"Will I miss out if I press the connect button under the other features because only this one has Redeem? But I only want Anti-spam, not backups right now".